README.md

```diff
@@ -140,6 +140,15 @@ See all possible Heroku flags:
 
     heroku help create
 
+In order to successfully push app to heroku, following environment variables must be set:
+
+* HOST
```

Comment:
This one is not necessary anymore, after https://github.com/thoughtbot/suspenders/commit/f6f4869f9dbad9104597bc4380330e8311f75d08.